pkg/pillar/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/Microsoft/hcsshim v0.8.7 // indirect
 	github.com/VictorLowther/godmi v0.0.0-20190311134151-270258a8252d // indirect
 	github.com/aws/aws-sdk-go v1.27.1
-	github.com/containerd/containerd v1.3.0
+	github.com/containerd/containerd v1.3.3
```

Comment:
Why is this PR pulling in a new version of containerd? By accident? Separate PR to make it easier to track?

Reply:
Hmm.. Did not notice it.. I guess a by-product of "go proto-vendor" ??

Reply:
Rolled back these unrelated containerd changes